node/benchmarks/relay_server.js

```diff
@@ -76,6 +76,8 @@ function RelayServer(opts) {
     });
     self.relay.handler = ServiceProxy({
         channel: self.relay,
+        totalRpsLimit: 30000,
+        defaultServiceRpsLimit: 30000,
```

Comment:
Infinity is a number in JavaScript and probably better conveys the spirit of this configuration.